Recalculating Datapoints failing in published container

**Describe the bug**

Recalculation of datasets is failing with the following exception:

```
2024-05-09 12:36:18,835 ERROR [io.hyp.too.hor.svc.AlertingServiceImpl] (executor-thread-11) Recalculation failed: java.lang.InternalError: java.lang.reflect.InvocationTargetException
	at com.oracle.truffle.runtime.ModulesSupport.loadModulesSupportLibrary(ModulesSupport.java:171)
	at com.oracle.truffle.runtime.ModulesSupport.<clinit>(ModulesSupport.java:60)
	at com.oracle.truffle.runtime.hotspot.HotSpotTruffleRuntimeAccess.createRuntime(HotSpotTruffleRuntimeAccess.java:82)
	at com.oracle.truffle.runtime.hotspot.HotSpotTruffleRuntimeAccess.getRuntime(HotSpotTruffleRuntimeAccess.java:73)
	at com.oracle.truffle.api.Truffle.createRuntime(Truffle.java:145)
	at com.oracle.truffle.api.Truffle$1.run(Truffle.java:176)
	at com.oracle.truffle.api.Truffle$1.run(Truffle.java:174)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)
	at com.oracle.truffle.api.Truffle.initRuntime(Truffle.java:174)
	at com.oracle.truffle.api.Truffle.<clinit>(Truffle.java:63)
	at com.oracle.truffle.runtime.enterprise.EnterpriseTruffle.supportsEnterpriseExtensions(stripped:21)
	at com.oracle.truffle.polyglot.enterprise.EnterprisePolyglotImpl.getPriority(stripped:518)
	at java.base/java.util.Comparator.lambda$comparing$77a9974f$1(Comparator.java:473)
	at java.base/java.util.TimSort.countRunAndMakeAscending(TimSort.java:355)
	at java.base/java.util.TimSort.sort(TimSort.java:220)
	at java.base/java.util.Arrays.sort(Arrays.java:1307)
	at java.base/java.util.ArrayList.sort(ArrayList.java:1721)
	at java.base/java.util.Collections.sort(Collections.java:179)
	at org.graalvm.polyglot.Engine.loadAndValidateProviders(Engine.java:1636)
	at org.graalvm.polyglot.Engine$1.run(Engine.java:1712)
	at org.graalvm.polyglot.Engine$1.run(Engine.java:1707)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)
	at org.graalvm.polyglot.Engine.initEngineImpl(Engine.java:1707)
	at org.graalvm.polyglot.Engine$ImplHolder.<clinit>(Engine.java:190)
	at org.graalvm.polyglot.Engine.getImpl(Engine.java:442)
	at org.graalvm.polyglot.Engine$Builder.build(Engine.java:740)
	at io.hyperfoil.tools.horreum.svc.Util.createContext(Util.java:600)
	at io.hyperfoil.tools.horreum.svc.Util.evaluateWithCombinationFunction(Util.java:563)
	at io.hyperfoil.tools.horreum.svc.AlertingServiceImpl.emitDatapoints(AlertingServiceImpl.java:453)
	at io.hyperfoil.tools.horreum.svc.AlertingServiceImpl.recalculateDatapointsForDataset(AlertingServiceImpl.java:324)
	at io.hyperfoil.tools.horreum.svc.AlertingServiceImpl.recalculateForDataset(AlertingServiceImpl.java:1055)
	at io.hyperfoil.tools.horreum.svc.AlertingServiceImpl_Subclass.recalculateForDataset$$superforward(Unknown Source)
	at io.hyperfoil.tools.horreum.svc.AlertingServiceImpl_Subclass$$function$$23.apply(Unknown Source)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:73)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext$NextAroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:97)
	at io.hyperfoil.tools.horreum.server.RolesInterceptor.intercept(RolesInterceptor.java:70)
	at io.hyperfoil.tools.horreum.server.RolesInterceptor_Bean.intercept(Unknown Source)
	at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:42)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:70)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:62)
	at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorBase.invokeInOurTx(TransactionalInterceptorBase.java:136)
	at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorBase.invokeInOurTx(TransactionalInterceptorBase.java:107)
	at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorRequiresNew.doIntercept(TransactionalInterceptorRequiresNew.java:41)
	at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorBase.intercept(TransactionalInterceptorBase.java:61)
	at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorRequiresNew.intercept(TransactionalInterceptorRequiresNew.java:32)
	at io.quarkus.narayana.jta.runtime.interceptor.TransactionalInterceptorRequiresNew_Bean.intercept(Unknown Source)
	at io.quarkus.arc.impl.InterceptorInvocation.invoke(InterceptorInvocation.java:42)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.perform(AroundInvokeInvocationContext.java:30)
	at io.quarkus.arc.impl.InvocationContexts.performAroundInvoke(InvocationContexts.java:27)
	at io.hyperfoil.tools.horreum.svc.AlertingServiceImpl_Subclass.recalculateForDataset(Unknown Source)
	at io.hyperfoil.tools.horreum.svc.AlertingServiceImpl.startRecalculation(AlertingServiceImpl.java:1007)
	at io.hyperfoil.tools.horreum.svc.AlertingServiceImpl.lambda$recalculateDatapoints$51(AlertingServiceImpl.java:968)
	at io.smallrye.context.impl.wrappers.SlowContextualRunnable.run(SlowContextualRunnable.java:19)
	at io.hyperfoil.tools.horreum.svc.Util.lambda$wrapForBlockingExecution$3(Util.java:510)
	at io.hyperfoil.tools.horreum.bus.TaskQueue.executeOrAdd(BlockingTaskDispatcher.java:59)
	at io.hyperfoil.tools.horreum.bus.BlockingTaskDispatcher.lambda$executeForTest$0(BlockingTaskDispatcher.java:30)
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$1(ContextImpl.java:191)
	at io.vertx.core.impl.ContextInternal.dispatch(ContextInternal.java:279)
	at io.vertx.core.impl.ContextImpl.lambda$internalExecuteBlocking$2(ContextImpl.java:210)
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76)
	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:582)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at com.oracle.truffle.runtime.ModulesSupport.loadModulesSupportLibrary(ModulesSupport.java:164)
	... 66 more
Caused by: java.nio.file.AccessDeniedException: /home/jboss/.cache
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:90)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:106)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixFileSystemProvider.createDirectory(UnixFileSystemProvider.java:397)
	at java.base/java.nio.file.Files.createDirectory(Files.java:700)
	at java.base/java.nio.file.Files.createAndCheckIsDirectory(Files.java:807)
	at java.base/java.nio.file.Files.createDirectories(Files.java:793)
	at com.oracle.truffle.polyglot.InternalResourceCache.installResource(InternalResourceCache.java:212)
	at com.oracle.truffle.polyglot.InternalResourceCache.installRuntimeResource(InternalResourceCache.java:173)
	... 71 more
```

**To Reproduce**

Recalculate datasets

**Version**

0.13.0
```
git rev-parse HEAD
```

**Java**

What is the version of Java ?

```
java -version
```

**PostgreSQL**

What is the version of PostgreSQL ?

```
psql --version
```

**OS**

Which Operating System (OS) is used ?

**Browser**

Which is the version of your browser ?

**Tip**

Use \`\`\` before and after the text to keep the output as is.

--- a/horreum-backend/src/main/docker/Dockerfile.jvm.base
+++ b/horreum-backend/src/main/docker/Dockerfile.jvm.base
@@ -9,7 +9,7 @@ RUN INSTALL_PKGS="python3 python3-devel python3-setuptools python3-pip python3-p
     rpm -V $INSTALL_PKGS && \
     rpm -e --nodeps redhat-logos-httpd && \
     microdnf -y clean all --enablerepo='*'
-USER 1001
+USER 185
 RUN pip install git+https://github.com/datastax-labs/hunter
 COPY src/main/resources/horreum.sh /deployments/
 COPY src/main/resources/k8s-setup.sh /deployments/

